### PR TITLE
Validation: Added Missing Alt Text

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text and missing descriptions.

### CODE REVIEW

File: aspnet/signalr/overview/older-versions/scaleout-with-redis.md [hunk 1]
```diff
@@ -112,15 +112,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![](scaleout-with-redis/_static/image5.png)
+![Screenshot of the Add Roles and Features Wizard with the Server Roles  and Web Socket Protocol options being highlighted.](scaleout-with-redis/_static/image5.png)
```
Comment (kturetz) [inline aspnet/signalr/overview/older-versions/scaleout-with-redis.md:115]: Remove extra space after "Roles"

File: aspnet/signalr/overview/older-versions/scaleout-with-redis.md [hunk 2]
```diff
@@ -112,15 +112,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
...
 
 Also include the Management Service (listed under "Management Tools").
 
-![](scaleout-with-redis/_static/image6.png)
+![Screenshot of the Add Roles and Features Wizard with the Serer Roles and I I S Management Scripts and Tools options being highlighted.](scaleout-with-redis/_static/image6.png)
```
Comment (kturetz) [inline aspnet/signalr/overview/older-versions/scaleout-with-redis.md:119]: "Serer" or "Server"?

File: aspnet/signalr/overview/older-versions/scaleout-with-redis.md [hunk 3]
```diff
@@ -112,15 +112,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
...
 
 **Install Web Deploy 3.0.** When you run IIS Manager, it will prompt you to install Microsoft Web Platform, or you can [download the installer](/iis/publish/using-web-deploy/microsoft-web-deploy-v3-readme). In the Platform Installer, search for Web Deploy and install Web Deploy 3.0
 
-![](scaleout-with-redis/_static/image7.png)
+![Screenshot of the WEb Platform Installer 4 point 5 search results screen with the Web Deploy 3 point 0 option being highlighted.](scaleout-with-redis/_static/image7.png)
```
Comment (kturetz) [inline aspnet/signalr/overview/older-versions/scaleout-with-redis.md:123]: "WEb" or "Web"?

File: aspnet/signalr/overview/older-versions/scaleout-with-sql-server.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 uid: signalr/overview/older-versions/scaleout-with-sql-server
-title: "SignalR Scaleout with SQL Server (SignalR 1.x) | Microsoft Docs"
+title: SignalR Scaleout with SQL Server (SignalR 1.x) | Microsoft Docs
 author: bradygaster
-description: ""
+description: Describes the SgnalR Scaleout with SQL Server application and provides prerequisites an overview on how to create and run the application.
```
Comment (kturetz) [inline aspnet/signalr/overview/older-versions/scaleout-with-sql-server.md:5]: "and overview"?

**Comment by dereklh7** [comment]
@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**